api/src/pcapi/core/educational/models.py

```diff
@@ -161,14 +161,21 @@ def mark_as_used_by_institute(self) -> None:
         self.status = EducationalBookingStatus.USED_BY_INSTITUTE
 
     def mark_as_refused(self) -> None:
-        from pcapi.core.bookings.models import BookingCancellationReasons
+        from pcapi.core.bookings import models as bookings_models
 
         if self.status == EducationalBookingStatus.USED_BY_INSTITUTE:
             raise exceptions.EducationalBookingNotRefusable()
 
+        if (
+            self.booking.status != bookings_models.BookingStatus.PENDING
+            and self.booking.cancellationLimitDate is not None
```

Comment:
Pas besoin du `is not None` non? car elle sera forcément égale à 15j avant date de l'évènement min?

Reply:
Ah oui bien vu, dans le cadre d'un EAC c'est forcément renseigné